install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
We no longer support bash?

Reply:
sh exist in all unix-like systems, but usually it ix link to bash.